ethanschreur/kids-krafts
# Sellers can edit orders

**What does this PR do?**
- This pull request allows sellers to edit customer orders

**Description of the task to be completed** 
- Add a page where you can edit individual order information
- Present that order's info in inputs that can be changed
- Have a save button that saves the edits

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/orders (log in if you have to)
- Fill out the "Add Orders" form and enter it
- Click on the added order in the "Orders" table
- Edit that order's info
- Press the save button
- Check that the updated info is shown
- Go back to the orders page and check that the updated info is shown
